packages/backend/server/src/core/workspaces/resolvers/team.ts

```diff
@@ -0,0 +1,235 @@
+import { Logger } from '@nestjs/common';
+import {
+  Args,
+  Mutation,
+  Parent,
+  ResolveField,
+  Resolver,
+} from '@nestjs/graphql';
+import { PrismaClient, WorkspaceMemberStatus } from '@prisma/client';
+
+import {
+  MailService,
+  NotInSpace,
+  RequestMutex,
+  SpaceNotFound,
+  TooManyRequest,
+  UserNotFound,
+} from '../../../fundamentals';
+import { CurrentUser } from '../../auth';
+import { FeatureManagementService, FeatureType } from '../../features';
+import { Permission, PermissionService } from '../../permission';
+import { QuotaManagementService } from '../../quota';
+import { UserService } from '../../user';
+import {
+  InviteResult,
+  TeamWorkspaceConfigType,
+  UpdateTeamWorkspaceConfigInput,
+  WorkspaceType,
+} from '../types';
+import { WorkspaceResolver } from './workspace';
+
+/**
+ * Workspace team resolver
+ * Public apis rate limit: 10 req/m
+ * Other rate limit: 120 req/m
+ */
+@Resolver(() => WorkspaceType)
+export class TeamWorkspaceResolver {
+  private readonly logger = new Logger(TeamWorkspaceResolver.name);
+
+  constructor(
+    private readonly mailer: MailService,
+    private readonly prisma: PrismaClient,
+    private readonly permissions: PermissionService,
+    private readonly users: UserService,
+    private readonly feature: FeatureManagementService,
+    private readonly quota: QuotaManagementService,
+    private readonly mutex: RequestMutex,
+    private readonly workspace: WorkspaceResolver
+  ) {}
+
+  @ResolveField(() => TeamWorkspaceConfigType, {
+    description: 'Team workspace config',
+    complexity: 2,
+    nullable: true,
+  })
+  async teamConfig(@Parent() workspace: WorkspaceType) {
```

Comment:
quota?

Reply:
this will make the logic straightforward for frontend, there is no need to detect whether a workspace is team one before fetching it's quota